app/services/foreman_bootdisk/renderer.rb

```diff
@@ -4,15 +4,30 @@ module ForemanBootdisk
   class Renderer
     include ::Foreman::Renderer
     include Rails.application.routes.url_helpers
+    delegate :logger, :to => :Rails
```

Comment:
Creating plugin logger and initializing it in the engine.rb as a class attribute.